bayesian/api_v1.py

```diff
@@ -1213,9 +1213,16 @@ def get(modified_date, ecosystem=None):
         except ValueError:
             msg = 'Invalid datetime specified. Please specify in YYYYMMDD format'
             raise HTTPError(400, msg)
-        getcve = CveByDateEcosystemUtils(modified_date, ecosystem)
+        page = request.args.get('page', '1')
+        if not page.isdigit() or page == '0':
+            raise HTTPError(400, error="Page parameter should be a positive integer")
+        ecosystem = request.args.get('ecosystem', None)
+        getcve = CveByDateEcosystemUtils(int(page), modified_date, ecosystem)
         result = getcve.get_cves_by_date() if not ecosystem else getcve.get_cves_by_date_ecosystem()
-        return jsonify(result), 200
+        headers = {}
+        if result['count'] == 25:
```

Comment:
Rather than hard-coding this value, it should be part of config params (openshift template) and should be able to be set via saas-analytics PRs